dongamboard/app/src/main/java/com/study/dongamboard/PostActivity.kt

```diff
@@ -122,6 +134,32 @@ class PostActivity : AppCompatActivity() {
         inputMethodManager.hideSoftInputFromWindow(activity.window.decorView.applicationWindowToken, 0)
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.menu_post, menu)
+        return true
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when (item.itemId) {
+            R.id.miUpdatePost->{
+                CoroutineScope(Dispatchers.IO).launch {
+                    val intent = Intent(applicationContext, PostUpdateActivity::class.java)
+                    intent.putExtra("postData", post)
+                    startActivity(intent)
+                    reloadPost()
+                }
+            }
+            R.id.miDeletePost->{
+                CoroutineScope(Dispatchers.IO).launch {
+                    ApiObject.getRetrofitAPIService.deletePost(post.id)
+                    // 댓글 삭제 요청 추가
```

Comment:
추후 할 작업은 TODO: 형식으로 작성해주세요!
```suggestion
                    // TODO: 댓글 삭제 요청 추가
```

Reply:
넵!